datadog_checks_base/datadog_checks/base/utils/tracing.py

```diff
@@ -2,8 +2,12 @@
 # All rights reserved
 # Licensed under Simplified BSD License (see LICENSE)
 import functools
+import os
 
-from ddtrace import tracer
+try:
+    from ddtrace import patch_all, tracer
+except ImportError:
+    tracer = None
```

Comment:
Can we add a log line here to denote that tracer is none?

Reply:
This will add a log line to everyone not using the tracer, I think it's noisy in most of the use cases, and we should try limiting log line that are not used